com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDDynamicResolutionPlatformCapabilities.cs

```diff
@@ -4,29 +4,19 @@ namespace UnityEngine.Rendering.HighDefinition
 {
     public static class HDDynamicResolutionPlatformCapabilities
     {
-        public enum Flag
-        {
-            DLSSDetected
-        }
+        /// <summary>
+        /// True if the render pipeline detected DLSS capable platform. False otherwise.
+        /// </summary>
+        public static bool DLSSDetected { get { return m_DLSSDetected; } }
 
-        public static bool GetFlag(Flag flag)
-        {
-            return (s_FeatureFlags & (1 << (int)flag)) != 0;
-        }
+        /// <summary>
+        /// User only variable, to control wether DLSS is on or off programmatically / from user code..
```

Comment:
I think user code here is implicit given that docs is for user, I'd just say 

"Variable used to control whether DLSS is on or off"

Reply:
Removed user control of this. In a separate PR we must figure out how to override at runtime DRS Settings in general (right now we can't)
For now added camera overrides, but still is not ideal (user might want to turn it all off)